client/settings/fraud-protection/advanced-settings/rule-toggle.tsx

```diff
@@ -33,16 +35,17 @@ const radioOptions = [
 ];
 
 const helpTextMapping = {
-	unchecked: __(
-		'When enabled, the payment method will not be charged until you review and approve the transaction.'
-	),
+	unchecked: __( 'When enabled, the payment method will be blocked.' ),
```

Comment:
The payment will be blocked, not the method.

Reply:
Fixed on [43ba3f7](https://github.com/Automattic/woocommerce-payments/pull/6275/commits/43ba3f7b7d22b12fd1be6d6913638671b90280be).